x-pack/elastic-agent/pkg/agent/application/info/agent_info.go

```diff
@@ -44,3 +46,13 @@ func ForceNewAgentInfo() (*AgentInfo, error) {
 func (i *AgentInfo) AgentID() string {
 	return i.agentID
 }
+
+// Version returns the version for this Agent.
+func (*AgentInfo) Version() string {
```

Comment:
we use release.Version and release.Snapshot throughout the code, so maybe we can unify the usage of these where it's possible in a followup

Reply:
Yeah I would like to. I only had to add this here for testing purposes as I needed a way to mock the version for the unit tests.